addon/core/schema.ts

```diff
@@ -231,19 +231,21 @@ export function renderRdfaAware({
   delete clone.resource;
   delete clone.__rdfaId;
   delete clone.rdfaNodeType;
-  const renderAttrs = renderRdfaAttrs(renderable);
+  const renderAttrs = { ...clone, ...renderRdfaAttrs(renderable) };
   let { property } = renderAttrs;
-  if (renderable.attrs.datatype === 'rdf:XMLLiteral') {
+  let datatype = renderAttrs.datatype || clone.datatype;
```

Comment:
Is the second part (`clone.datatype`) necessary here?, as `clone` is already included in `renderAttrs` right?

Reply:
Part of the problems I'm working around here is that datatype can be set to `null` in the return of `renderRdfaAttrs`, which means that `renderAttrs.datatype` is null, even if `clone.datatype` is defined. Maybe worth a comment in the code?

Reply:
Done